sapphire/analysis/calibration.py

```diff
@@ -86,6 +91,67 @@ def determine_detector_timing_offset(dt, dz=0):
     return detector_offset, rchi2
 
 
+class station_timing_offsets(object):
+    """Determine the timing offsets between stations"""
+
+    def __init__(self, data, station_id, ref_station_id):
+        """
+
+        :param data: pytables HDF5 file with timedelta tables
+        :param station_id: station for which to determine station offsets
+        :param ref_station_id: station id of reference station
+        """
+        self.data = data
+        self.station_id = station_id
+        self.ref_station_id = ref_station_id
+        self.station = Station(station_id)
+        self.ref_station = Station(ref_station_id)
+        self.cluster = HiSPARCStations([station_id, ref_station_id])
+        self.r, _, self.dz = self.cluster.calc_rphiz_for_stations(0, 1)
```

Comment:
This `r` should depend on the timestamp/dates for which the offset is determined, so for each split determine `r`.